docs/src/main/asciidoc/cdi-reference.adoc

```diff
@@ -734,6 +734,29 @@ class SharedService {
 <2> `@Lock(Lock.Type.READ)` overrides the value specified at class level. It means that any number of clients can invoke the method concurrently, unless the bean instance is locked by `@Lock(Lock.Type.WRITE)`. 
 <3> You can also specify the "wait time". If it's not possible to acquire the lock in the given time a `LockException` is thrown.
 
+=== Repeatable interceptor bindings
+
+Quarkus supports `@Repeatable` interceptor binding annotations, if they are declared on methods.
+For example:
+
+[source,java]
+----
+@ApplicationScoped
+class AllEncompassingService {
+  @Cached(cacheName = "foo") // <1>
```

Comment:
AsciiDoc actually understands `// <1>` and removes the `//`. Much better, since my IDE knows it's a Java code in that `.adoc` file and gives me syntax highlight and errors and everything -- I don't know why noone else does this! :-)

Reply:
That's good to know, thanks for the info!